androidApp/src/main/java/press/IntentReceiverActivity.kt

```diff
@@ -0,0 +1,20 @@
+package press
+
+import android.app.Activity
+import android.os.Bundle
+import press.editor.EditorActivity
+
+class IntentReceiverActivity : Activity() {
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    processIntent()
+
+  }
+
+  private fun processIntent() {
```

Comment:
Let's also inline this function. This whole class has just one job, no point in extracting this to a separate function.